account_invoice_import_invoice2data/models/account_invoice_import_config.py

```diff
@@ -0,0 +1,21 @@
+# coding: utf-8
+# Copyright (C) 2021 Essent <http://www.essent.be>
+# @author Stefan Rijnhart <stefan.rijnhart@dynapps.be>
+# @author Robin Conjour <r.conjour@essent.be>
+# @author Pieter Paulussen <pieter.paulussen@dynapps.be>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+from openerp import models, fields
+
+
+class AccountInvoiceImportConfig(models.Model):
+    _inherit = 'account.invoice.import.config'
+
+    # Extend invoice_line_methods with `product_mapping`
```

Comment:
Please use `selection_add=` instead of overwriting the whole selection.

Reply:
Done